assignments/array-methods.js

```diff
@@ -58,28 +58,59 @@ const runners = [
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs. Combine both the first and last names and populate a new array called `fullNames`. This array will contain just strings.
 let fullNames = [];
+
+runners.forEach(function(name){
+return fullNames.push(`${name.first_name} ${name.last_name}`)
+});
 console.log(fullNames);
 
+
+
+
+
+
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runners' first names in uppercase because the director BECAME DRUNK WITH POWER. Populate an array called `firstNamesAllCaps`. This array will contain just strings.
-let firstNamesAllCaps = [];
+let firstNamesAllCaps = runners.map(function(item){
+  return item.first_name.toUpperCase();
+});
+
 console.log(firstNamesAllCaps);
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue. We need a filtered version of the runners array, containing only those runners with large sized shirts so they can choose a different size. This will be an array of objects.
-let runnersLargeSizeShirt = [];
+let runnersLargeSizeShirt = runners.filter (function(item) {
```

Comment:
Whenever you call a function or a method, make sure the parentheses are right next to the word like `runners.filter(function(item) {`. Most of the time, Javascript will understand what you mean, but it's good practice to remove the space '' '